### ISSUE TITLE
Filters should be designed to be inclusive to mobile users and blind users who use screen readers

### ISSUE THREAD

**Issue by marcusrelacion** [AIGAdc/dcdesignweek.org-11ty#9]
https://docs.google.com/document/d/1P-M_Zq1p9FbTdfl6vcA9qjrvg1qqtRzbFELp1hvnO6s/edit#heading=h.mwxidltosebh

**Comment by marcusrelacion** [comment]
@joshkimux going to close this ticket for now as there have been a number of improvements made regarding the event filters.
